lib/core.js

```diff
@@ -119,7 +119,13 @@ Zone.prototype = {
   onZoneCreated: function () {},
   afterTask: function () {},
   enqueueTask: function () {},
-  dequeueTask: function () {}
+  dequeueTask: function () {},
+  addEventListener: function () {
+    return this['_addEventListener'].apply(this, arguments);
```

Comment:
See https://github.com/angular/zone.js/pull/171/files#diff-50e3aa130a4f97a42ee2cf111c7b1d9dR92.

The `this` binding from the `addEventListener` call is used for the `apply` on `global.zone.addEventListener`. I don't see how else you could implement this as `patchEventTargetMethods` can be called on different objects when `EventTarget` isn't implemented.